README.md

```diff
@@ -103,12 +103,42 @@ Public Docker image located at [link](https://hub.docker.com/repository/docker/b
 
 ### Usage
 
-You now can POST queries to `http://<HOST>:3000/v1/query`.  
+You now can POST queries to `http://<HOST>:3000/v1/query`.
 
 Query Examples can be found [here](/examples).
 
 
-### Test with local SmartAPI spec
+### Testing with Alternate SmartAPI Specs (local or hosted)
+
+#### Using `API_OVERRIDE=true`
+
+You may configure a set of API IDs to override from local files or URLs.
+
+If the environment variable `API_OVERRIDE=true` is set (e.g. `API_OVERRIDE=true npm run debug --workspace=@biothings-explorer/bte-trapi`), then `/config/smartapi_overrides.json` is checked at server start and overrides are applied, as well as during subsequent `smartapi_specs.json` updates.
+
+Override files are expected to be in yaml format, with local files expected in the `data` folder. If overrides are specified with IDs not in the current SmartAPI spec, they will be appended as new API hits with a log warning.
+
+You may also set `only_overrides` to `true` in the config to remove all other APIs and keep only the specified overrides.
+
+Example:
+
+Replace the latest MyGene.info API with a specific revision, and the MyChem.info API with a local test version:
+
+```JSON
+{
+  "conf": {
+    "only_overrides": false
+  },
+  "apis": {
+    "59dce17363dce279d389100834e43648": "https://raw.githubusercontent.com/NCATS-Tangerine/translator-api-registry/8b36f46d59c82d19b5cba40421a6ca9c2ed62e6b/mygene.info/openapi_full.yml",
+    "8f08d1446e0bb9c2b323713ce83e2bd3": "file:///mychem_test.yaml"
```

Comment:
@tokebe since we allow to use `file://` URL string, we should allow any arbitrary file path. You may find this useful:

https://stackoverflow.com/questions/20619488/how-to-convert-local-file-path-to-a-file-url-safely-in-node-js

Reply:
I've added support for arbitrary file paths, while keeping `file:///` expecting the data directory. Essentially, there are 3 options now:
- URL which GETs a yaml file
- `file:///` which uses the data folder as the root directory
- An arbitrary file path e.g. `/some/arbitrary/path/file.yaml` which does not use the data folder as the root directory

Please let me know if you'd like me to remove the special behavior for `file:///` so I can update the README with this.

Reply:
all three supported options sound good to me.